controllers/org_rbac_controller.go

```diff
@@ -57,13 +62,17 @@ func (r *OrganizationRBACReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	}
 
 	var errs []error
+	if nsNoAccess, nsNoAccessExists := ns.Labels[LabelNsNoAccess]; nsNoAccessExists && (strings.ToLower(nsNoAccess) == "true") {
```

Comment:
Final nitpick: You don't really need to check if the label exists. If it doesn't `nsNoAccess` is `""` which isn't `true`. What I'd rather check is if `ns.Labels` is `nil`. (I know it never is in this case otherwise `org` would be `""`, but I'd still check it in case someone changes the code down the line)

Reply:
Moved the logic to a new function (as with the `getOrganization` function). This should resolve this nitpick